### DISCUSSION TITLE
"rotation" problem on export

### DISCUSSION THREAD

**Discussion by carnussien** [mifi/lossless-cut#661] [Q&A]
Hello,
On preview i can apply rotation on the video and see the rotated video.
BUT "export" does not keep the rotation.
The video keeps its initial orientation.
Can you help?
Thanks
PS: I forgot to add that i am using a lossless-cut appimage for linux (ubuntu)..

**Comment by mifi** [comment]
if you can provide a short sample file then I can look into it

**Comment by masvil** [comment]
Same here, video attached.
[rotation-test.zip](https://github.com/mifi/lossless-cut/files/5990791/rotation-test.zip)

**Reply by mifi** [reply]
Thanks for your sample. I just realized that MKV doesn't support lossless rotation (the rotate metadata). Correct me if I'm wrong!
If you try with an MP4 file it will work.

**Comment by masvil** [comment]
Exporting to MP4 instead of MKV does the trick, if you don't care changing the format.

@mifi please evaluate to make LC alerting the user in case of an attempt to rotate a MKV. Maybe a suggestion to change the output format to MP4 (or something) could help even more to improve the UX.

**Comment by edmundlaugasson** [comment]
Using Losslesscut 3.65.0, which is current latest version. Same problem. Using MP4 file (H.264, 720x1080, yuv420p) but does not export as rotated. Kindly suggest to add apply button to save rotation effect and exit from rotation preview. Then we can be sure, that rotation has been really applied.

**Reply by mifi** [reply]
Rotation is temporarily broken in 3.65.0, you can try latest nightly build or rollback. see https://github.com/mifi/lossless-cut/commit/af9fe16bb4986c11b819c6fef1df1de410450b8f
